app/lib/plugins/settings/index.js

```diff
@@ -1,5 +1,4 @@
 import get from './get'
 import validate from './validate'
-import add from './add'
 
-export default { get, validate, add }
```

Comment:
But neither of those actually add the settings to the config. Does this work for you while testing with a plugin that defines some settings? Maybe I'm missing something.